test/integration/schema/index.js

```diff
@@ -1048,6 +1048,24 @@ module.exports = function(knex) {
               expect(exists).to.equal(true);
             });
         });
+
+        describe('sqlite only', function() {
+          if (
+            !knex ||
+            !knex.client ||
+            !/sqlite3/i.test(knex.client.driverName)
+          ) {
+            return Promise.resolve();
+          }
+
+          it('checks whether a column exists without being case sensitive, resolving with a boolean', function() {
```

Comment:
Could you use arrow function instead of `function()` and async/await instead of `then` chain?

Reply:
Sure. Not a problem even if not consistent with the rest of the file ?

Reply:
yeah, we want to eventually convert everything